scripts/release_scripts/cut_release_or_hotfix_branch_test.py

```diff
@@ -325,23 +325,21 @@ def test_missing_release_version(self):
             sys, 'argv', ['cut_release_or_hotfix_branch.py'])
         with args_swap, self.assertRaisesRegexp(
             Exception, 'ERROR: A "release_version" arg must be specified.'):
-            cut_release_or_hotfix_branch.execute_branch_cut()
+            cut_release_or_hotfix_branch.main()
 
     def test_exception_is_raised_if_travis_is_failing(self):
         def mock_input():
             return 'n'
 
         input_swap = self.swap(python_utils, 'INPUT', mock_input)
-        args_swap = self.swap(
-            sys, 'argv',
-            ['cut_release_or_hotfix_branch.py', '--release_version=1.2.3'])
         with self.verify_local_repo_swap, self.verify_branch_name_swap:
             with self.verify_target_branch_swap:
                 with self.verify_target_version_swap, self.open_tab_swap:
                     with self.get_remote_alias_swap, self.check_call_swap:
-                        with input_swap, args_swap, self.assertRaises(
+                        with input_swap, self.assertRaises(
```

Comment:
Everywhere, use assertRaisesRegexp and include an error message, to verify that the exception is being raised in the correct context.

Reply:
(I've also added this to #8423 as a lint check request.)

Reply:
A `system.exit` is raised here, so there is not specific message to check.

Reply:
Where is the SystemExit raised from? I think you should be able to add a message to it explaining what went wrong...

Reply:
Updated.